src/commands/server/start.ts

```diff
@@ -186,6 +186,23 @@ export default class Start extends Command {
 
   checkPlatformCompatibility(flags: any) {
     // matrix checks
+    if (flags.installer === 'operator' && flags['che-operator-cr-yaml']) {
+      let msg = ''
+      msg += flags['plugin-registry-url'] ? '\t--plugin-registry-url' : ''
+      msg += flags['devfile-registry-url'] ? '\t--devfile-registry-url' : ''
+      msg += flags['postgres-pvc-storage-class-name'] ? '\t--postgres-pvc-storage-class-name' : ''
+      msg += flags['workspace-pvc-storage-class-name'] ? '\t--workspace-pvc-storage-class-name' : ''
+      msg += flags['self-signed-cert'] ? '\t--self-signed-cert' : ''
+      msg += flags['os-oauth'] ? '\t--os-oauth' : ''
+      msg += flags.tls ? '\t--ls' : ''
+      msg += flags.cheimage ? '\t--cheimage' : ''
```

Comment:
![Screenshot_20200224_092914](https://user-images.githubusercontent.com/5887312/75134836-2caa9680-56e8-11ea-93b0-b32ff6d5e23f.png)

It's not critical but cheimage is printed even when user does not specify it.

Reply:
![Screenshot_20200224_093135](https://user-images.githubusercontent.com/5887312/75134922-798e6d00-56e8-11ea-91e7-5a3e34a3a624.png)